web3.js

```diff
@@ -18,16 +18,18 @@ function load(opts) {
   if (null === opts || (undefined !== opts && 'object' !== typeof opts)) {
     throw new TypeError('Expecting options to be an object.')
   } else if ('object' !== typeof opts) {
-    // eslint-disable-next-line no-param-reassign
+    // eslint-disable no-param-reassign
     opts = {}
   }
-
   if (!opts.provider) {
     if (rc.web3 && rc.web3.provider) {
-      // eslint-disable-next-line no-param-reassign
+      opts.provider = rc.web3.provider
+    }
+    if (rc.web3 && rc.web3.network_id) {
```

Comment:
Isn't this redundant? `getProvider` returns the providers using `network_id` if `network_id` is present, otherwise `web3.provider`.

Reply:
it covers all flavors of `.ararc`s . if only a web3.provider is present, it sets that.  if only a network_id is present, it sets that.  if both are present, it sets network_id.  it is the second scenario that wasn't covered by former code.

Reply:
👍